### DISCUSSION TITLE
[.NET7] HyperV setup not working

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#10447] [Q&A] [answered]
I'm trying to setup my .NET7 UNO env as per [here](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=windows) by running `uno-check --pre`.

At some point it keeps complaining that the HyperV setup could not be completed and I needed to be reboot my PC - which I did. However, it still is complaining (see screenshot).

What am I doing wrong?

<img width="675" alt="image" src="https://user-images.githubusercontent.com/26179242/201981379-85d47e7c-db5a-40c5-905c-a99c5499d4ee.png">

**Comment by jeromelaban** [comment]
The Hyper-V check is done for Android acceleration. You may find more details by running with the [verbosity flag](https://platform.uno/docs/articles/external/uno.check/doc/using-uno-check.html#troubleshooting).

**Reply by DierkDroth** [reply]
@jeromelaban ok here is what I see applying `--verbose`

Does this help analyzing the problem?

<img width="675" alt="image" src="https://user-images.githubusercontent.com/26179242/202622943-4a80846d-709c-456f-800d-99aad594e1ed.png">

**Reply by jeromelaban** [reply]
The command is the right one, but you may be in an environment that does not support Hyper-V. Are you running in a virtual machine? Which edition of Windows are you using?

**Reply by DierkDroth** [reply]
This is a regular DELL Window 11 laptop. Pretty much the latest - purchased new as of earlier this year.

Please let me know if you needed anything else

**Reply by jeromelaban** [reply]
Yes, please tell me which edition of windows you are you using. In all cases, you most likely won't need this if you're not doing android development.

**Reply by DierkDroth** [reply]
@jeromelaban is this what you needed (screenshot below + system info)?

Yes, I understand that's for Android 'only'. However:
- I will have to get into iOS + Android soon
- at the end of the day I wanted to assist you guys to improve your product ;-)

Device name	Unusual
Processor	11th Gen Intel(R) Core(TM) i9-11900H @ 2.50GHz   2.50 GHz
Installed RAM	32.0 GB (31.7 GB usable)
Device ID	F47CE19D-4E25-4DF1-9AC4-18C63C2590F6
Product ID	00342-20794-06522-AAOEM
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

Edition	Windows 11 Home
Version	22H2
Installed on	‎9/‎21/‎2022
OS build	22621.819
Experience	Windows Feature Experience Pack 1000.22636.1000.0


<img width="1196" alt="image" src="https://user-images.githubusercontent.com/26179242/202762156-6ed96974-cb46-4553-a942-4b5ccde4e21b.png">

**Reply by jeromelaban** [reply]
Thanks. Hyper-V is not available for Windows 11 Home, which is why the check cannot work. We'll change the validation for that.

**Reply by DierkDroth** [reply]
Thanks for clarification @jeromelaban

**Comment by DierkDroth** [comment, marked as answer]
By @jeromelaban 
> Hyper-V is not available for Windows 11 Home, which is why the check cannot work.
